src/uct/api/uct.h

```diff
@@ -1303,22 +1309,40 @@ typedef struct uct_md_mem_attr {
 
```

Comment:
```
"Note that the field mask is
     * populated upon return from uct_md_mem_query and not set by user."
```
is this text correct?

Reply:
Thanks for the catch @bureddy It's not true. I'll address this.